packages/admin/config/filament.php

```diff
@@ -205,6 +205,7 @@
             'groups' => [
                 'are_collapsible' => true,
             ],
+            'width' => '320px'
```

Comment:
IMO the default should be null and set in the view instead, so we have the flexibility to customise it ourselves internally if we wish

Reply:
updated with latest commit